mantidimaging/gui/windows/operations/presenter.py

```diff
@@ -346,3 +352,18 @@ def init_crop_coords(self, roi_field: QLineEdit):
         y = min(self.stack.presenter.images.data[0].shape[1], 200)
         crop_string = ", ".join(["0", "0", str(y), str(x)])
         roi_field.setText(crop_string)
+
+    def _show_negative_values_error(self, negative_stacks: List[StackVisualiserView]):
+        """
+        Shows information on the view and in the log about negative values in the output.
+        :param negative_stacks: A list of stacks with negative values in the data.
+        """
+        names = [stack.name for stack in negative_stacks]
+        self.view.show_error_dialog(f"Negative values found in stack(s) {', '.join(names)}. See log for more details.")
```

Comment:
Would it be possible to show this when a negative value is detected in the preview. Even if it only detects negatives on the current projection I think it would still be useful.